pkg/tenant/db/members.go

```diff
@@ -85,6 +79,10 @@ func CreateTenantMember(ctx context.Context, member *Member) (err error) {
 		member.ID = ulids.New()
 	}
 
+	if ulids.IsZero(member.TenantID) {
```

Comment:
I added the validation check for Tenant ID separately because it's currently not available in the claims, so adding it to the validation method would cause `CreateMember` tests to fail on the server-side. If Tenant ID is added to claims, then this will be updated.